core/client/fs/src/test/java/alluxio/client/file/options/OutStreamOptionsTest.java

```diff
@@ -119,14 +119,14 @@ public void fields() throws Exception {
     int writeTier = random.nextInt();
     WriteType writeType = WriteType.NONE;
 
+    mConf.set(PropertyKey.USER_FILE_CREATE_TTL_ACTION, "FREE");
```

Comment:
I noticed below you're only checking the ttlAction instead of both. Is that intentional? Can we set the ttl here as well?